clients/crystal/src/backup_restore.cr

```diff
@@ -0,0 +1,39 @@
+module MoanaClient
+  class Backup
```

Comment:
Create single class `Backups` with methods `backup`, `restore`. Delete can be added later.